django/db/migrations/writer.py

```diff
@@ -503,6 +509,8 @@ def serialize(cls, value):
 
 MIGRATION_TEMPLATE = """\
 # -*- coding: utf-8 -*-
+# Generated by Django %(version)s
+# on %(timestamp)s
```

Comment:
I don't think using the development version of Django is the case to optimize for.